downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -613,6 +613,11 @@ $= future<int> result = start name();
 
 #### CLI
 
+#### EDI tool
+
+- Introduced support for field length constraints (min/max), enhancing validation capabilities for EDI data fields.
+- Added support for EDIFACT to Ballerina schema conversion, facilitating smoother integrations and automations.
```

Comment:
I prefer 
```suggestion
- Added support for EDIFACT to Ballerina schema conversion. 
```

Reply:
also wouldn't it be better to mention the new command/command option name (perhaps using an example)?

Reply:
Addressed with [a67adcb](https://github.com/ballerina-platform/ballerina-dev-website/pull/8938/commits/a67adcbb103b6abe051f066eabbe84bf450a4422).